include/aws/http/connection.h

```diff
@@ -221,11 +228,44 @@ struct aws_http_client_connection_options {
     bool manual_window_management;
 };
 
+/**
+ * Options for changing the settings for an HTTP/2 connection.
+ */
+struct aws_http2_change_settings_options {
```

Comment:
I don't know we really need this structure or not... Will this make the logic too complicated? Without this, the parameters for aws_http2_connection_change_settings will be complicated...

Reply:
Yeah, these should probably just be function arguments.
`_options` structs are good for functions where you think it's very likely that we'll add more arguments in the future. This seems like it has all that it needs